contents/blog/the-posthog-array-1-9-0.md

```diff
@@ -0,0 +1,120 @@
+* We fixed long href inserts - the href [can now go up to 2048 characters](https://github.com/PostHog/posthog/pull/1027) before truncation. Someone must have had some funky urls going on...
+* [We prevented intermittent issues with yarn build](https://github.com/PostHog/posthog/pull/1026)
+* We [fixed a bug](https://github.com/PostHog/posthog/pull/1021) that caused cohorts to fail when actions were deleted
+* We [solved a problem](https://github.com/PostHog/posthog/pull/980) with comparing trend sessions distribution
+* We [added a limit to number of returned entities for breakdowns](https://github.com/PostHog/posthog/pull/1008) so queries don't time out
+* We [created a fix](https://github.com/PostHog/posthog/pull/1013) for an issue with heartbeats
+* We [made it clearer](https://github.com/PostHog/posthog/pull/1014) that PostHog SaaS users are on the latest version
+* We [slashed CPU consumption for VSCode](https://github.com/PostHog/posthog/pull/1007) by excluding a folder
+* Generated a [performance improvement for element stats](https://github.com/PostHog/posthog/pull/991)
+* We [stopped giving way too many decimal points](https://github.com/PostHog/posthog/pull/984) on our graphs!
+* Trends page [UX improvement](https://github.com/PostHog/posthog/pull/919)
+* [Improved filtering](https://github.com/PostHog/posthog/pull/986) on elements
+* We fixed [a race condition](https://github.com/PostHog/posthog/pull/973/commits/953af2326dff94e8ae1d75cd6ea0fc2c64567857)
+* [We don't rely](https://github.com/PostHog/posthog/pull/949) on \$ to separate PostHog's events
+* We [removed the redundant math selector](https://github.com/PostHog/posthog/pull/950) on funnels - it didn't do anything!
+* [Django upgraded to 3.0.7](https://github.com/PostHog/posthog/pull/932)
+* We [made HTTPS work locally](https://github.com/PostHog/posthog/pull/910) - we had lots of community issues raised, so that should make it easier to get started with!
+* We [improved the setup overlay layout](https://github.com/PostHog/posthog/pull/904)
+* We [sped up the events endpoint](https://github.com/PostHog/posthog/pull/903) by just hitting the current week's partitions
+* We solved a problem [with temporary tokens](https://github.com/PostHog/posthog/pull/909)
+* We added [webpack HMR](https://github.com/PostHog/posthog/pull/878) and hashes to chunk filenames. (#878)
+
+
+## Favorite issue
+
+### [Blog header design](https://github.com/PostHog/posthog.com/issues/128)
+
+We had a lot of fun this week thinking about how the PostHog brand looks and feels, and welcomed Lottie - our new designer - to the team!
+
+## Weekly round up
+
+From our own blog:
+
+* [How we raised $3M for an open source project](/blog/raising-3m-for-os) - we hope this helps other open source projects.
```

Comment:
This could be a top-of-email announcement

Reply:
done